src/harness/unittests/tsserverProjectSystem.ts

```diff
@@ -17,6 +17,30 @@ namespace ts.projectSystem {
         })
     };
 
+    const typeMapList = {
+        path: <Path>"/typeMapList.json",
+        content: JSON.stringify({
+            "jquery": {
+                // jquery files can have names like "jquery-1.10.2.min.js" (or "jquery.intellisense.js")
+                "match": "/jquery(-(\\.\\d+)+)?(\\.intellisense)?(\\.min)?\\.js$",

```

Comment:
This don't match `jquery-1.10.2.min.js`, matches something like `jquery-.10.2.min.js`.